### ISSUE TITLE
Can not index file. SK Error: error opening input file

### ISSUE THREAD

**Issue by javanyou** [rockbruno/swiftshield#86]
-- Indexing ButtonNode.swift --
ERROR: Could not index ButtonNode.swift, aborting. SK Error: error opening input file '/Users/xxxx/.../UI\ Components/ButtonNode.swift' (No such file or directory)

I think this maybe a blank in path. 
I have test the file path in error info, it's can be touched, so the file path in program is right.

Waiting for someone help

**Comment by kevinhassan** [comment]
Yes I already got this issue linked to whitespace in the path #83.
